examples/configuration.nix

```diff
@@ -172,6 +172,15 @@
   # and electrs data directory, enable
   # services.backups.with-bulk-data = true;
 
+  ### JOINMARKET
+  # Enable this module to allow using JoinMarket's user interactive scripts (including
```

Comment:
I'd mention here that joinmarket has priviledged bitcoind access.

Reply:
Implemented in `fixup! module: note joinmarket's full access to bitcoind`